script.js

```diff
@@ -0,0 +1,449 @@
+    43: {code: 'KeyZ', key1: 'z', key3: 'я'},
+    44: {code: 'KeyX', key1: 'x', key3: 'ч'},
+    45: {code: 'KeyC', key1: 'c', key3: 'с'},
+    46: {code: 'KeyV', key1: 'v', key3: 'м'},
+    47: {code: 'KeyB', key1: 'b', key3: 'и'},
+    48: {code: 'KeyN', key1: 'n', key3: 'т'},
+    49: {code: 'KeyM', key1: 'm', key3: 'ь'},
+    50: {code: 'Comma', key1: ',', key2: '<', key3: 'б'},
+    51: {code: 'Period', key1: '.', key2: '>', key3: 'ю'},
+    52: {code: 'Slash', key1: '/', key2: '?', key3: '.', key4: ','},
+    53: {code: 'ArrowUp', name: 'ᐃ', class: 'arrow-up'},
+    54: {code: 'ShiftRight', name: 'Shift', class: 'shift-right', location: 2},
+    55: {code: 'ControlLeft', name: 'Ctrl', class: 'ctrl-left', location: 1},
+    56: {code: 'MetaLeft', name: 'Win', class: 'win', location: 1},
+    57: {code: 'AltLeft', name: 'Alt', class: 'alt-left', location: 1},
+    58: {code: 'Space', name: '', class: 'space'},
+    59: {code: 'AltRight', name: 'Alt', class: 'alt-right', location: 2},
+    60: {code: 'ArrowLeft', name: 'ᐊ', class: 'arrow-left'},
+    61: {code: 'ArrowDown', name: 'ᐁ', class: 'arrow-down'},
+    62: {code: 'ArrowRight', name: 'ᐅ', class: 'arrow-right'},
+    63: {code: 'ControlRight', name: 'Ctrl', class: 'ctrl-left', location: 2},
+}
+
+let main = document.createElement('main');
+main.className = 'main';
+document.body.prepend(main);
+
+// Input___________________________________
+let result = document.createElement('div');
+result.className = 'result';
+main.prepend(result);
+let textarea = document.createElement('textarea');
+textarea.className = 'textarea';
+textarea.autofocus = 'true';
+result.prepend(textarea);
+
+// Keyboard_________________________________
+let keyboard = document.createElement('div');
+keyboard.id = 'keyboard';
+main.append(keyboard);
+keyboard.prepend(getRowAll());
+
+// Readme___________________________________
+let readme = document.createElement('div');
+readme.className = 'readme';
+readme.insertAdjacentHTML('beforeend', '<p>CtrlRight + AltRight = En-Ru</p>');
+readme.insertAdjacentHTML('beforeend', '<p>OS Windows</p>');
+main.append(readme);
+
+// Отображение клавиш______________________________________________________
+let keyList = document.getElementById('keyboard').querySelectorAll('.key');
+let ctrlAlt = sessionStorage.getItem('ctrlAlt');
+for(let k = 0; k < keyList.length; k++) {
+    let key = keyList[k];
+    let keyKeyboardEn = keyboardEn[k];
+    if('class' in keyKeyboardEn ) {
+        key.classList.add(keyKeyboardEn['class']);
+    }
+    key.setAttribute('code', keyKeyboardEn['code']);
+    if('key1' in keyKeyboardEn) {
+        key.setAttribute('key1', keyKeyboardEn['key1']);
+        if(ctrlAlt === 'false' || ctrlAlt === null) {
+            key.insertAdjacentHTML('beforeend', `<span>${keyKeyboardEn['key1']}</span>`);
+        }
+        if(ctrlAlt === 'true') {
+            if(k > 0 && k <= 12 || k === 27) {

```

Comment:
magic numbers. what is 12? example: const KEYCODE_CLEAR= 12; if (k === KEYCODE_CLEAR) {}